entity-framework/core/providers/index.md

```diff
@@ -1,17 +1,77 @@
-﻿---
+---
 title: Database Providers - EF Core
 author: rowanmiller
 ms.author: divega
-
-ms.date: 10/27/2016
-
+ms.date: 2/23/2018
 ms.assetid: 14fffb6c-a687-4881-a094-af4a1359a296
 ms.technology: entity-framework-core
-
 uid: core/providers/index
 ---
+
 # Database Providers
 
-Entity Framework Core uses a provider model to allow EF to be used to access many different databases. Some concepts are common to most databases, and are included in the primary EF Core components. Such concepts include expressing queries in LINQ, transactions, and tacking changes to objects once they are loaded from the database. Some concepts are specific to a particular provider. For example, the SQL Server provider allows you to configure memory-optimized tables (a feature specific to SQL Server). Other concepts are specific to a class of providers. For example, EF Core providers for relational databases build on the common `Microsoft.EntityFrameworkCore.Relational` library, which provides APIs for configuring table and column mappings, foreign key constraints, etc.
+Entity Framework Core can access many different databases through plug-in libraries called database providers.
+
+## Current providers
+> [!IMPORTANT]  
+> EF Core providers are built by a variety of sources. Not all providers are maintained as part of the Entity Framework Core project. When considering a provider, be sure to evaluate quality, licensing, support, etc. to ensure they meet your requirements. Also make sure you review each provider's documentation for detailed version compatibility information.
+
+| NuGet Package                                                                                                     | Supported database engines | Maintainer / Vendor                                                           | Notes                       | Useful links                                                                                                                                                              |
+|:------------------------------------------------------------------------------------------------------------------|:---------------------------|:------------------------------------------------------------------------------|:----------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
This markdown table is read-optimized. It should be easier to review it using two screens :Smile: